content/docs/09.administrator-guide/02.deployment/07.google-vm.md

```diff
@@ -0,0 +1,197 @@
+2. Go to `Networking` tab, and click on `Add a Network`.
+3. In the New Network section, add an appropriate name like `Kestra VM`, and put your VM instance's external IP address in the Network.
+4. Click on `Done` in the section.
+5. Click on `Save` on the page.
+
+![db_connections](/docs/administrator-guide/deployment/gcp-vm/db_connections.png)
+
+![db_add_a_network](/docs/administrator-guide/deployment/gcp-vm/db_add_a_network.png)
+
+![db_save_network](/docs/administrator-guide/deployment/gcp-vm/db_save_network.png)
+
+**Create database user**
+
+1. Go to the database overview page, and click on `Users` from the left side navigation menu.
+2. Click on `Add User Account`.
+3. Put an appropriate username and password, and click on `Add`.
+
+![db_users](/docs/administrator-guide/deployment/gcp-vm/db_users.png)
+
+![db_user_creation](/docs/administrator-guide/deployment/gcp-vm/db_user_creation.png)
+
+**Create Kestra database**
+
+1. Go to the database overview page, and click on `Databases` from the left side navigation menu.
+2. Click on `Create Database`.
+3. Put an appropriate database name, and click on `Create`.
+
+**Update Kestra configuration**
+
+In the docker-compose configuration, edit the `datasources` property of the Kestra service in the following way:
+
+```yaml
+datasources:
+    postgres:
+    url: jdbc:postgresql://<your-db-external-endpoint>:5432/<db_name>
+    driverClassName: org.postgresql.Driver
+    username: <your-username>
+    password: <your-password>
+```
+
+Because you now use RDS service, you don't need the postgres service anymore. Remove it from the `docker-compose.yml` file.
+
+In order for the changes to take effect, restart the docker services with `docker compose restart` or `docker compose up -d`.
+
+
+## Step 5: Use AWS S3 for storage
+
+By default, internal storage is implemented using the local file system. This section will guide you on how to change the storage backend to Cloud Storage to ensure a more reliable, durable and scalable storage.
+
+1. Go to the Cloud Storage console and create a bucket.
+2. Go to IAM and select `Service Accounts` from the left side navigation menu.
+3. On the Service Accounts page, click on `Create Service Account` on the top of the page.
+4. Put appropriate Service account name and Service account description, and grant the service account `Storage Admin` access. Click Done.
+5. On the Service Accounts page, click on the newly created service account.
+6. On the newly created service account page, go to `Keys` tab on the top of the page, and click on `Add Key`. From the dropdown, select `Create New Key`.
+7. Select Key type as `JSON` and click on `Create`. The JSON key file for the service account will get downloaded.
+8. We will be using the stringified JSON for our configuration. You can use the bash command `% cat <path_to_json_file> | jq '@json'` to generate stringified JSON.
+9. Edit Kestra storage configuration.
```

Comment:
```suggestion
Because you now use the Cloud SQL service, you don't need the Postgres Docker service anymore. Remove it from the `docker-compose.yml` file.

In order for the changes to take effect, restart the docker services with `docker compose restart` or `docker compose up -d`.


## Step 5: Use AWS S3 for storage

By default, internal storage is implemented using the local file system. This section will guide you on how to change the storage backend to Cloud Storage to ensure more reliable, durable, and scalable storage.

1. Go to the Cloud Storage console and create a bucket.
2. Go to IAM and select `Service Accounts` from the left-side navigation menu.
3. On the Service Accounts page, click on `Create Service Account` at the top of the page.
4. Put the appropriate Service account name and Service account description, and grant the service account `Storage Admin` access. Click Done.
5. On the Service Accounts page, click on the newly created service account.
6. On the newly created service account page, go to the `Keys` tab at the top of the page and click on `Add Key.` From the dropdown, select `Create New Key`.
7. Select the Key type as `JSON` and click on `Create`. The JSON key file for the service account will get downloaded.
8. We will be using the stringified JSON for our configuration. You can use the bash command `% cat <path_to_json_file> | jq '@json'` to generate stringified JSON.
9. Edit Kestra storage configuration.
```